open-models/serving/vertex_ai_text_generation_inference_gemma.ipynb

```diff
@@ -0,0 +1,794 @@
+{
```

Comment:
<p>I would suggest something like <code>To serve Gemma with Text Generation Inference (TGI) on Vertex AI, you start importing ...</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1064/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>